rules/license.bzl

```diff
@@ -67,24 +69,47 @@ _license = rule(
                   " This may be used to produce an index of OSS packages used by" +
                   " an applicatation.",
         ),
+        "package_url": attr.string(
+            doc = "The URL this instance of the package was download from." +
+                  " This may be used to produce an index of OSS packages used by" +
+                  " an applicatation.",
+        ),
```

Comment:
It's needlessly confusing to document this as being the location where the package was downloaded from while calling it `package_url` and referring to use in SBOM creation.  In SBOM creation, what people generally need is a package URL conforming to the [package URL spec](https://github.com/package-url/purl-spec), which is _not_ a download link.